adapters/avail/src/spec/mod.rs

```diff
@@ -0,0 +1,30 @@
+use sov_rollup_interface::da::DaSpec;
```

Comment:
Let's enforce documentation:
```
#![deny(missing_docs)]
#![doc = include_str!("../README.md")]
```
?

Reply:
Synced with @bkolad offline. @vibhurajeev, we're ok merging this as-is as long as docs and tests are added in a follow-up